metahtml/timestamp.py

```diff
@@ -383,7 +383,7 @@ def get_timestamp_published(html, url, **kwargs):
         ( 'stripes.com',                    '//span[@class="published_date"]' ),
         ( 'thediplomat.com',                '//span[@itemprop="datePublished"]' ),
         ( 'theintercept.com',               '//span[@class="PostByline-date"]' ),
-        ( 'theworldnews.net',               '//span[@class="news__time"]/text()'),
+        #( 'theworldnews.net',               '//span[@class="news__time"]/text()'),
```

Comment:
In general, you shouldn't have commented out code within a project.  It should just be deleted.